knees4bees/overlook
# Feature/iteration1

## What does this PR do? ##
- Completes iteration 1 on the project spec, finally!!
- Creates classes and associated tests for `Room`, `Booking`, `BookingsRepo`, `Customer`, and `helpers`
- Adds dashboard and past/current bookings info to the DOM
- Creates initial styling (hope to update later)

## How should it be tested? ##
1. First fetch and install my code:
  - Clone the repo (name it something else so it doesn't conflict with yours)
  - `npm install`
  - `git fetch`
  - `git checkout feature/iteration1`
  - in a second Terminal tab, `npm start`
2. Confirm that all tests are passing when you run `npm test`
3. In a browser, go to `http://localhost:8080/` and confirm that the needed data is being displayed. (Note that it says the user has "points" at the top but that's really a dollar amount in disguise w/ 100 points/dollar.)
4. Take a look at the class files (`Room.js`, `Customer.js`, `Booking.js`, `BookingsRepo.js`) and `helpers.js` and see if you spot anything that looks like a bad move or a code smell.

I'd also love your thoughts on any styling tweaks I could make later to make this look more profesh!
